src/atoms/Modal/withModalState.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { Consumer } from './ModalContext';
+
+const withModalState = (BaseComponent) => {
+  class ModalStateConsumer extends React.Component {
+    renderBaseComponent = (modalContext) => {
```

Comment:
I think you should add more specificity to this method. Maybe name it `renderBaseComponentWithModalProps`?

Reply:
Because visually in code i don't see difference between `return <BaseComponent />` and `renderBaseComponent`.

Reply:
@vorobeez fixed 👍